.devcontainer/devcontainer.json

```diff
@@ -1,7 +1,8 @@
 {
     "name": "Flux Core Python 3.6",
     "dockerFile": "Dockerfile",
-  
+    "context": "../",
```

Comment:
Excellent! This is what I didn't find, much better solution.

Reply:
For future us - the walk through guides aren't very detailed in what you can put in this file, but this guide (assuming you mostly know what the variables mean) is! I'll leave here for future us to find: https://containers.dev/implementors/json_reference/

Reply:
And you know what' really funny - I almost bought this domain back when .dev was new!! https://containers.dev/ Glad I didn't, I would have probably wasted it on something silly. :laughing: